### PR TITLE
Tidy-up after liquidity provision mechanics changes

### LINKED ISSUE #7188: Liquidity score not reset when fees accrued in a period are 0

### LINKED ISSUE #7189: LP orders outside PM price range but within LP price range not included in the liquidity score
Since LP orders outside PM price range but within LP price range were not included in the liquidity score, it meant that an LP could set things up so that majority all of their automatically deployed orders where outside PM bounds but within LP range (if it was wider than PM bounds) and then place a single limit order close to the mid. That would make their liquidity score very large, despite most of the volume being deployed outside PM range.

### PR DESCRIPTION
**Pull request by witgaw**
- Uncomment and tidy-up scenario 4 in`0035-LIQM-liquidity_monitoring.feature` (commented out as part of 6955 work)
- Add integration test for 6956
- Fix 7188
- Fix 7189

Closes #7188 
Closes #7189

### CODE REVIEW

File: core/liquidity/supplied/engine.go [hunk 1]
```diff
@@ -121,18 +121,19 @@ func (e *Engine) CalculateLiquidityScore(
...
 	bSize := num.DecimalZero()
 	sSize := num.DecimalZero()
 	for _, o := range orders {
-		if o.Price.LT(lowerBound) || o.Price.GT(upperBound) {
+		if o.Price.LT(minLpPrice) || o.Price.GT(maxLpPrice) {
 			continue
 		}
-		prob := getProbabilityOfTrading(bestBid, bestAsk, minPMPrice.Original(), maxPMPrice.Original(), e.pot, o.Price.ToDecimal(), o.Side == types.SideBuy, e.minProbabilityOfTrading)
+		prob := num.DecimalZero()
+		// if orders is outside of price monitoring bounds then probability is set to 0.
```
Comment (jeremyletang) [inline core/liquidity/supplied/engine.go:133]:
```suggestion
		// if order is outside of price monitoring bounds then probability is set to 0.
```

**Review by jeremyletang — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,7 +12,8 @@
 - [](https://github.com/vegaprotocol/vega/issues/xxxx) -
 
 ### 🐛 Fixes
-- [](https://github.com/vegaprotocol/vega/issues/xxxx) -
```
Comment (davidsiska-vega) [inline CHANGELOG.md:15]: This probably should stay as template for the next guy?

**Review by davidsiska-vega — APPROVED** [review]
Happy with the feature test.

File: CHANGELOG.md (hunk 2 above)
Comment (witgaw) [reply CHANGELOG.md:15]: I think it's fine, the next guy will just grab one of the existing ones, that's how it usually goes.

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**